packages/frontend/tests/acceptance/dashboard/accessibility-test.js

```diff
@@ -25,4 +25,12 @@ module('Acceptance | dashboard accessibility', function (hooks) {
     assert.dom('#ember-a11y-refocus-skip-link').exists();
     assert.dom('#ember-a11y-refocus-skip-link').hasProperty('tabIndex', 0);
   });
+
+  test('meets a11y standards in calendar view', async function (assert) {
```

Comment:
i'm seeing this test case being added via https://github.com/ilios/frontend/pull/8668 as well.

shouldn't be a problem, but thought i mention in in case we run into merge conflicts down the road.